doc/onlinehelp.rst

```diff
@@ -1,29 +1,92 @@
-:orphan:
-
 ###########
 Online Help
 ###########
 
 The **Online Help** pane provides a built-in web browser to explore dynamically generated Python documentation on installed modules—including your own—rendered by a ``pydoc`` server running in the background.
```

Comment:
I think we should include a link to the Pydoc module in Python standard library documentation. Otherwise, I think this sentence

> rendered by a ``pydoc`` server running in the background.

says very little to people.

Reply:
Will do.
https://docs.python.org/3/library/pydoc.html